metrics/serve.go

```diff
@@ -177,19 +177,19 @@ func handleDoubleHalveMode(metricCount, metricLength, metricCycle, seriesCycle i
 }
 
 func handleGradualChangeMode(metricCount, metricLength, metricCycle, seriesCycle int, labelKeys, labelValues []string, seriesChangeRate, maxSeriesCount, minSeriesCount int, changeSeriesTick *time.Ticker, updateNotify chan struct{}) {
```

Comment:
left a comment on slack as well, but this function still needs to take in a pointer for the `seriesCount` so that when it modifies the value the other go routines know the correct value for `seriesCount`